transformers/tokenization_utils.py

```diff
@@ -512,8 +512,9 @@ def add_tokens(self, new_tokens):
         to_add_tokens = []
         for token in new_tokens:
             assert isinstance(token, six.string_types)
-            if (token != self.unk_token and
-                    self.convert_tokens_to_ids(token) == self.convert_tokens_to_ids(self.unk_token)):
+            if token != self.unk_token and \
```

Comment:
PEP8 highly recommends not using backslashes in Python code because whitespace to the right of the backslash can break the script on a change that is invisible to the user.  () are preferred.

Reply:
👍 want to fix it and I can merge?